app/Http/Controllers/User/BookmarksController.php

```diff
@@ -164,7 +164,10 @@ public function store(Request $request)
             return $this->setStatusCode(422)->replyWithError($invalidBookmark);
         }
 
-        $bookmark_id = Bookmark::create($request->all())->id;
+        $new_bookmark = $request->all();
+        $new_bookmark['verse_sequence'] = $request->verse_sequence ?? (int) $request->verse_start;
```

Comment:
is this trying to cast a string to an int? If yes, then what will happen if verse_start is "3a"?